### DISCUSSION TITLE
Question:

### DISCUSSION THREAD

**Discussion by GStudiosX2** [FunkinCrew/Funkin#1501] [Q&A]
.

**Comment by TentaRJ** [comment]
It's open source.

**Comment by TentaRJ** [comment]
[Look at the license](https://github.com/ninjamuffin99/Funkin/blob/master/LICENSE)

**Comment by TentaRJ** [comment]
This is the license for this repository. You can check it yourself.

**Comment by TentaRJ** [comment]
Talk to him about it then. Mention him. If you do, you are just gonna waste your time because it is the same license that is on the home page of the repository.

If you don't think it is the same license, then prove me wrong.

**Comment by TootNoot** [comment]
ninja muffin has said that he is fine with fangames

**Comment by TootNoot** [comment]
back to what tenta says, there is a license saying that you are allowed to use the assets, the game is open source so really you can use whatevers in the repository. Creators of a public repository set the license themselves and know what they're setting it for. There is no need to bother ninja muffin for something that has already been answered by the license

**Comment by TootNoot** [comment]
The license is your approval.
